src/CMakeLists.txt

```diff
@@ -327,12 +327,11 @@ IF(NOT USE_BUNDLED_ZLIB)
 	IF(ZLIB_FOUND)
 		LIST(APPEND LIBGIT2_SYSTEM_INCLUDES ${ZLIB_INCLUDE_DIRS})
 		LIST(APPEND LIBGIT2_LIBS ${ZLIB_LIBRARIES})
-		IF(APPLE OR CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
-			LIST(APPEND LIBGIT2_LIBS "z")
-			LIST(APPEND LIBGIT2_PC_LIBS "-lz")
-		ELSE()
-			LIST(APPEND LIBGIT2_PC_REQUIRES "zlib")
-		ENDIF()
+        	IF(APPLE OR CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
+            		LIST(APPEND LIBGIT2_PC_LIBS "-lz")
+        	ELSE()
+            		LIST(APPEND LIBGIT2_PC_REQUIRES "zlib")
+        	ENDIF()
```

Comment:
Sorry to be a nag, but there's some leading spaces now in front of the tabs. If you convert that to all-tabs then we're ready to go!

Reply:
Tabs versus spaces can be a pain! Hopefully all is now good.